flink-ai-flow/ai_flow/api/workflow_operation.py

```diff
@@ -79,13 +85,63 @@ def submit_workflow(workflow_name: Text = None) -> WorkflowInfo:
     workflow = translator.translate(graph=current_graph(), project_context=current_project_context())
     _apply_full_info_to_workflow(entry_module_path, workflow)
     current_graph().clear_graph()
+    response = get_ai_flow_client().register_workflow(name=workflow_name,
```

Comment:
Do we want to make sure that submit_workflow_to_scheduler returns successfully before registering the workflow to metadata service?